client/components/spinner-button/style.scss

```diff
@@ -0,0 +1,4 @@
+.spinner-button__spinner {
+	float: right;
+	padding: 8px;
+}
```

Comment:
Files should end with a newline. I recommend installing an [EditorConfig](http://editorconfig.org/) plugin for your editor, for which we [specify a configuration](https://github.com/Automattic/wp-calypso/blob/master/.editorconfig).